### DISCUSSION TITLE
[Skill Calculator] Show XP difference between current and target experience.

### DISCUSSION THREAD

**Discussion by ekohvakka** [runelite/runelite#16163] [Ideas]
The title says it all pretty much. It would be helpful to see the XP difference between the chosen current and target experience values somewhere near the top of the skill calculator.

**Comment by Nightfirecat** [comment]
That is shown by hovering the target experience field.
![Screenshot_20230115_110429](https://user-images.githubusercontent.com/2199511/212561685-660e12b4-8908-42a1-8d3e-df474f5ae16e.png)

**Reply by ekohvakka** [reply]
Thanks. Makes sense that this already exists since it's so simple, but I've been looking for this from the client for around 2 years.
Cheers.
